rero/rero-ils-ui
# routing: redirect after saving a resource

## Why are you opening this PR?

Implements task 116 of US985.
Adds a routing type to redirect to document detail view after saving an item.

## How to test?

Needs PR on ng-core.
Create or edit an item and check redirection after saving it.

## Code review check list
